app/views/unattended/provisioning_templates/provision/atomic_kickstart_default.erb

```diff
@@ -65,10 +65,11 @@ reboot
 cp -vf /tmp/*pre*log /tmp/sysimage/root/
 %end
 
+<%= snippet('remote_execution_ssh_keys_kickstart') %>
```

Comment:
This may be just me, but I would consider moving it to after `rootpw`. That means you keep all the post sections together and also the authorization together. (This is relevant for all kickstarts, not just Atomic but I won't repeat it everywhere.)

Reply:
Yeah, I know for fact that order is insignificant in kickstart, you can start with `%post` or whatever. Will move, sure.